packages/manager/modules/account-sidebar/src/controller.js

```diff
@@ -26,11 +26,17 @@ export default class OvhManagerAccountSidebarCtrl {
       this.sidebarExpand = false;
     });
 
-    const unregisterListener = this.$rootScope.$on('ovh-chatbot:enable', () => {
-      this.hasChatbot = true;
-      this.links = this.getLinks();
-      unregisterListener();
-    });
+    const unregisterListener = this.$rootScope.$on(
```

Comment:
it looks like you are adding arguments to the ovh-chatbot:enable event but i don't see that you are updating and passing new arguments where the events are fired, is it normal ?

Reply:
Here : https://github.com/ovh/manager/pull/6498/files#diff-4ca5b53f5c5ac2a716e2fc229ab16cb90edc260bd08463e2c01120839825d72eR46-R50